processors/tenantidprocessor/tenantidprocessor_test.go

```diff
@@ -506,16 +506,16 @@ func initResource1(r pcommon.Resource) {
 }
 
 func initResourceAttributes1(dest pcommon.Map) {
-	dest.UpsertString("resource-attr", "resource-attr-val-1")
+	dest.PutString("resource-attr", "resource-attr-val-1")
```

Comment:
So `PutString` replaces both `Insert` and `UpsertString`?

Reply:
Yeah, both Insert + Upsert have been removed as of v0.61.0. If you want Insert behavior it seems like you have to check if the key already exists since Put no longer handles that.